modules/repository/hooks.go

```diff
@@ -112,15 +111,27 @@ func CheckDelegateHooks(repoPath string) ([]string, error) {
 		newHookPath := filepath.Join(hookDir, hookName+".d", "gitea")
 
 		cont := false
-		if !com.IsExist(oldHookPath) {
+		isExist, err := util.IsExist(oldHookPath)
+		if err != nil {
+			results = append(results, fmt.Sprintf("unable to check if %s exists. Error: %v", oldHookPath, err))
+		}
+		if !isExist {
```

Comment:
```if err == nil && if !isExist {```

Reply:
```suggestion
		if err == nil && !isExist {
```

Reply:
This is small fry btw - getting multiple logs here is not a serious issue.